boruta/boruta_py.py

```diff
@@ -544,3 +548,6 @@ def _print_results(self, dec_reg, _iter, flag):
             result = '\n'.join([x[0] + '\t' + x[1] for x in zip(cols, content)])
             output = "\n\nBorutaPy finished running.\n\n" + result
         print(output)
+
+    def _is_lightgbm(self):
```

Comment:
what's the point of this function? can you pls just change line 188 to:
`self._is_lightgbm = 'lightgbm' in str(type(self.estimator))`
and then I'll merge this.

Reply:
and delete this function
thanks

Reply:
Done, thank you to you!